documentation/specs/static-graph.md

```diff
@@ -125,6 +118,9 @@ This is a proposal for what the public API may look like:
 
             // All project nodes in the graph.
             IReadOnlyCollection<ProjectGraphNode> ProjectNodes { get; }
+
+            // Build the graph with the given target, if any
+            BuildResult Build(string target = null);
```

Comment:
True, gotten ahead of myself. Though it could be a nice convenience method, just like Project and ProjectInstance have Build methods, which internally send themselves to a BuildManager.

Reply:
Good point. There are probably a ton of convenience overloads we should add in general. Probably towards the end through when we're polishing :)